Storage/src/Lifecycle.php

```diff
@@ -139,6 +139,80 @@ public function addDeleteRule(array $condition)
         return $this;
     }
 
+    /**
+     * Adds a AbortIncompleteMultipartUpload lifecycle rule to Object Lifecycle.
+     *
+     * Example:
+     * ```
+     * $lifecycle->addAbortIncompleteMultipartUploadRule([
```

Comment:
We added a snippet here, can we add a snippet test?
Also a unit test?

Reply:
Sorry, I should've added it before. Added in c99b2b4a43076a2e52226b94600f2c5886aff9da